rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] These two variables can also be modified hence you'll require getters and setters to achieve this: `attr_accessor :date, :book, :person`